lib/src/indicators/slider.dart

```diff
@@ -0,0 +1,362 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:macos_ui/macos_ui.dart';
+
+const double _kSliderMinWidth = 100.0;
+const double _kSliderBorderRadius = 16.0;
+const double _kSliderHeight = 4.0;
+const double _kContinuousThumbSize = 20;
+const double _kOverallHeight = 20;
+const double _kTickWidth = 2.0;
+const double _kTickHeight = 8.0;
+
+const double _kDiscreteThumbWidth = 6.0;
+const double _kDiscreteThumbBorderRadius = 8;
+
+/// A slider is a horizontal track with a control, called a thumb,
+/// that people can adjust between a minimum and maximum value.
+///
+/// The slider doesn't maintain any state itself, instead the user is expected to
+/// update this widget with a new [value] whenever the slider changes.
+///
+/// ![Slider](https://developer.apple.com/design/human-interface-guidelines/images/intro/components/slider-intro-dark_2x.png
```

Comment:
I pulled this down locally, and it looks like the image doesn't come through

Reply:
Forgot the closing bracket 😅